src/applications/hca/reducer.js

```diff
@@ -24,26 +31,29 @@ const initialState = {
 export function hcaEnrollmentStatus(state = initialState, action) {
   switch (action.type) {
     case FETCH_ENROLLMENT_STATUS_STARTED:
-      return { ...initialState, isLoading: true };
+      return { ...state, isLoadingApplicationStatus: true };
```

Comment:
My first version of this, where this action always reset the state to the `initialState` may have been a bad idea, but it certainly introduced a bug now that there is more data tracked in the state.